--- udp.go	2018-04-17 11:13:24.000000000 +0200
+++ new_udp.go	2018-04-18 03:12:29.000000000 +0200
@@ -114,7 +114,7 @@
 func NewRUDP(conn net.PacketConn, logger log.Logger) *RUDP {
 	t := &RUDP{
 		conn:       conn,
-		nextPktID:  uint56(generator.Intn(maxUint56 + 1)),
+		nextPktID:  uint56(generator.Int63n(maxUint56 + 1)),
 		readEvents: make(chan *readEventDesc, maxReadEvents),
 		closedChan: make(chan struct{}),
 		doneChan:   make(chan struct{}),
