diff --git a/src/makefile b/src/makefile
index 9bc897f..beec2cb 100644
--- a/src/makefile
+++ b/src/makefile
@@ -35,7 +35,7 @@ OPTIMIZATION?=-O2
 WARNINGS=-Wall -Wstrict-prototypes -fwrapv
 DEBUG?= -g -ggdb
 REAL_CFLAGS=$(OPTIMIZATION) -fPIC -pthread $(CFLAGS) $(WARNINGS) $(DEBUG)
-REAL_LDFLAGS=$(LDFLAGS) -pthread
+REAL_LDFLAGS=$(LDFLAGS) -pthread -llog -landroid-glob
 
 DYLIBSUFFIX=so
 STLIBSUFFIX=a
